cmd/nitro/filehandler_test.go

```diff
@@ -19,12 +19,13 @@ import (
 func pollLogMessagesFromJSONFile(t *testing.T, path string, expected []string) ([]string, error) {
 	t.Helper()
 	var msgs []string
+	var err error
 Retry:
-	for i := 0; i < 20; i++ {
+	for i := 0; i < 30; i++ {
 		time.Sleep(20 * time.Millisecond)
-		msgs, err := readLogMessagesFromJSONFile(t, path)
+		msgs, err = readLogMessagesFromJSONFile(t, path)
 		if err != nil {
-			return []string{}, err
+			continue
```

Comment:
I think below you meant to `return msgs, err` if the last iteration still had an error. As-is it seems to always ignore the error.

Reply:
yes, you're right. Fixed.